src/services/auth/authService.ts

```
@@ -165,10 +174,36 @@ export async function signin(params: SigninParams): Promise<SigninResult> {
 
   const match = await bcrypt.compare(passcode, user.passcodeHash);
   if (!match) {
-    logger.warn("Signin: invalid passcode", { userId: user.id });
+    const failedAttempts = user.failedSigninAttempts + 1;
+    const isLockout = failedAttempts >= config.maxSigninAttempts;
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)